msprime/_msprimemodule.c

```diff
@@ -1445,7 +1468,8 @@ Simulator_parse_demographic_events(Simulator *self, PyObject *py_events)
     int err, population_id, source, destination;
     int is_population_parameter_change, is_migration_rate_change, is_mass_migration,
         is_population_split, is_admixture, is_symmetric_migration_rate_change,
-        is_simple_bottleneck, is_instantaneous_bottleneck, is_census_event;
+        is_simple_bottleneck, is_instantaneous_bottleneck, is_census_event,
+        is_population_activation;
```

Comment:
Let's standardise on ``activate_population_event``, and use this systematically. So, ``is_activate_population_event`` here.